setup.cfg

```diff
@@ -0,0 +1,8 @@
+[flake8]
+max-line-length = 119
+per-file-ignores =
+    # redefinition of unused 'current_actor_context'
```

Comment:
Imho we should not keep commented code unless it's a quickfix.

Reply:
Well those lines were just a hint why we ignore F811 (redefinition) and F401 (unused imports).